tico/experimental/quantization/algorithm/gptq/quantizer.py

```diff
@@ -50,176 +68,193 @@ def prepare(
         Overrides the forward method of the first LLaMA layer (layer 0) to capture the
          input required for calibration.
 
-        This method modifies the original forward pass of LLaMA layer 0 so that the
-         inputs used during inference are intercepted and recorded. These captured inputs
-        are then utilized to calibrate the quantization parameters for the GPTQ.
+        When the user calls `model(...)`, we intercept (and store) the inputs to that
+         layer, then raise an exception to stop the forward pass immediately. These
+        captured inputs are then utilized to calibrate the quantization parameters
+         for the GPTQ.
 
         Parameters:
-            model: The target PyTorch model.
-            args: Positional example inputs required for capturing graph.
-            kwargs: Keyword example inputs required for capturing graph.
+            model (torch.nn.Module): The target PyTorch model
+            args (Any, optional): Unused (kept for API compatibility)
+            kwargs (Dict[str, Any], optional): Unused (kept for API compatibility)
```

Comment:
How about adding an assert for this unused options?

Reply:
I think the function spec of `BaseQuantizer` should be fixed. Let me fix this as well in another PR.